e2e/component_filtering_test.go

```diff
@@ -0,0 +1,110 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/e2e-framework/klient"
+	"sigs.k8s.io/e2e-framework/klient/wait"
+	"sigs.k8s.io/e2e-framework/klient/wait/conditions"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+
+	spinapps_v1alpha1 "github.com/spinkube/spin-operator/api/v1alpha1"
+	"github.com/spinkube/spin-operator/e2e/helper"
+	"github.com/stretchr/testify/require"
+)
+
+// TestComponentFiltering checks that the operator and shim have support for running a subset of a Spin app's components
+func TestComponentFiltering(t *testing.T) {
+	var client klient.Client
+
+	// TODO: Use an image from a sample app in this repository
+	appImage := "ghcr.io/kate-goldenring/spin-operator/examples/spin-salutations:20241022-144454"
```

Comment:
The salutations app is checked in and in the apps workflow so I'm not quite sure why we don't yet have a package published for you to point at.

Reply:
The publish action failed so there is still nothing to point to https://github.com/spinkube/spin-operator/actions/runs/11500919778